.github/actions/security-hub-scan/action.yaml

```diff
@@ -0,0 +1,68 @@
+name: security-hub-scan
```

Comment:
Common pattern we have is that the action would be tested in this repo, so if you wouldnt mind implementing this getting called as part of the workflow/workflows (test/publish) that would be great.

Reply:
ok, I see other examples that have that in this repo that I can follow.

this is going to need a zarf package for a test, and ideas on how to support that?

Reply:
a zarf package is built and published as part of this repo to provide an example and facilitate the repo's CI being able to test itself. It is currently [podinfo](https://github.com/defenseunicorns/uds-common/pkgs/container/packages%2Fuds%2Fpodinfo) and in the [next release](https://github.com/defenseunicorns/uds-common/pull/218) will be updated to nginx to provide a more representative example

Reply:
I was able to use that local package to test this workflow.

Reply:
added tests, so I am resolving this